src/templates/en/2019/table_of_contents.html

```diff
@@ -56,15 +56,23 @@
 }
 
 .chapter .todo {
+<<<<<<< HEAD
```

Comment:
There seem to be merge conflicts committed here?

Reply:
Fixed.